en/core-api/credit-card.md

```diff
@@ -1,57 +1,87 @@
 <!-- TODO: explain it also accept Debit card -->
 <!-- TODO: explain when the card is checked for balance/limit, after 3DS -->
-# Core API Card Transaction Integration
+
+# Card Payment Integration
 <hr>
-One of the payment method offered by Midtrans is Card transaction. By using this payment method, customers will have the option to pay using credit card (or online-transaction-capable debit card) that is within Visa, MasterCard, JCB, or Amex network. Midtrans will also send real time notification when the customer complete the payment.
+<i>Card</i> payment is one of the payment methods offered by Midtrans. Using this payment method, customers can make payments using a Card (or online-transaction-capable debit card) within Visa, MasterCard, JCB, or Amex network. Midtrans sends real-time notification when the customer completes the payment.
```

Comment:
> a Card (or online-transaction-capable debit card)
hmm.. sounds a bit unnatural, because you changed `credit card` to `a Card` but still leave the `debit card`.

Reply:
Should it be "credit card or any online-transaction-capable debit card."? Please help me rephrase it.

Reply:
Yes that should work. Thanks!